docs/docs-content/clusters/palette-virtual-clusters/configure-oidc-virtual-cluster.md

```diff
@@ -0,0 +1,106 @@
+description: "Learn how to configure OIDC for Palette Virtual Clusters."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["clusters", "cluster groups", "virtual clusters"]
+---
+
+
+Before deploying a virtual cluster, you must set up OpenID Connect (OIDC) Identity Provider (IDP) for the cluster group the virtual cluster belongs to. As virtual clusters are deployed to that cluster group, OIDC information in the cluster group configuration will be inserted into the virtual cluster configuration. 
+
+
+## Prerequisites
+
+- A healthy host cluster that you will use to create a cluster group.
+- A cluster group to which you can add virtual clusters.
+- An issuer URL and client ID that you obtain from your identity provider.
+- Privileges sufficient to edit cluster group settings and a viewer role to access the generated kubeconfig. If you are deploying virtual clusters, you will need admin privileges.
+- [kubelogin](https://github.com/int128/kubelogin) installed. This is a kubectl plugin for Kubernetes OIDC authentication, also known as `kubectl` oidc-login.
+
+
+## Configure OIDC
+
+Use these steps to configure your cluster group with OIDC *before* creating a virtual cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. If a cluster group does not exist, you will need to create a host cluster and then create the cluster group. Refer to [Create and Manage Cluster Groups](../../clusters/cluster-groups/create-cluster-group.md) for guidance.
+
+3. If a cluster group exists, navigate to the left **Main Menu** and click on **Cluster Groups**.
+
+4. Select the cluster group to which you will deploy a virtual cluster. 
+
+5. In the **Host Clusters** tab that displays on the next page, click on **Settings**.
+
+6. In the slide panel that opens, select the **Settings** option. The cluster group YAML file displays.
+
+7. Locate the `vcluster.extraArgs` parameter section of the cluster group configuration file and uncomment the lines shown in the example.
+
+![Screenshot of the cluster group YAML showing oidc-related parameters to uncomment and update.](/clusters_palette-virtual-clusters_configure-vcluster-oidc.png)
+
+8. Update the `vcluster.extraArgs` section with your identity provider information as shown in example.
+
+    ```yaml
+    vcluster:
+      extraArgs:	
+      - --kube-apiserver-arg=”oidc-issuer-url=<client-url>"
+      - --kube-apiserver-arg="oidc-client-id=<client-id>"
+      - --kube-apiserver-arg="oidc-username-claim=email"
+      - --kube-apiserver-arg="oidc-groups-claim=groups"
+    ```
+
+9. If your identity provider requires a client secret, uncomment the `oidc-client-secret` parameter in the `clientConfig` section of cluster group YAML file, and add the client secret.
+
+    ```yaml
+    clientConfig:	
+      oidc-client-secret: secret-value
+    ```
+
+10. From the **User Menu**, switch to *App Mode* and click on **Virtual Clusters**.  
```

Comment:
```suggestion
10. From the **User Menu**, switch to **App Mode**.  From the left **Main Menu**, click on **Virtual Clusters**.  
```
I think we need to help a newer user understand this is in the left Main Menu. Personally, I would had broken this up into two steps, but that's just my personal preference, nothing wrong with you combining this as it's a pretty simple task.

Reply:
applied the fix.